pom.xml

```diff
@@ -134,8 +134,14 @@
       <dependency>
         <groupId>org.codehaus.plexus</groupId>
         <artifactId>plexus-utils</artifactId>
-        <version>3.0.22</version>
+        <version>3.1.0</version>
```

Comment:
Any specific reason not using latest version (3.2.1)?

Reply:
When I opened the pull request almost a year ago, 3.1.0 was the newest version at that time. The latest version (3.2.1) is from July 10 this year ;-)

Reply:
OK, I see. 🤦‍♂ 
I saw that you updated this dependency, great! However, I believe there was some workaround for v3.1.0 somewhere. Is that still needed with v3.2.1?